src/renderer/components/select/select.tsx

```diff
@@ -72,7 +72,7 @@ export class Select extends React.Component<SelectProps> {
     const { autoConvertOptions, options } = this.props;
 
     if (autoConvertOptions && Array.isArray(options)) {
-      return options.map(opt => {
+      return (options as any[]).map(opt => {
```

Comment:
why the need for this case, if the function above is `T | any` then anything should be accepted, right?

Reply:
It was TSC some complaining and not related to this PR (just dirty fix)

Reply:
this unrelated changes removed from this PR